src/bundles/config.js

```diff
@@ -34,7 +35,18 @@ bundle.selectApiUrl = createSelector(
 
 bundle.selectGatewayUrl = createSelector(
   'selectConfigObject',
-  (config) => getURLFromAddress('Gateway', config) || 'https://ipfs.io'
+  async (config) => {
+    if (!config) return null
+
+    const url = getURLFromAddress('Gateway', config)
+
+    // More info: https://github.com/ipfs-shipyard/ipfs-webui/issues/1490#issuecomment-671633602
+    if (!await checkIfGatewayUrlIsAcessible(url)) {
+      return 'https://ipfs.io'
```

Comment:
We hardcoded `https://ipfs.io` in [a few places](https://github.com/ipfs-shipyard/ipfs-webui/search?l=JavaScript&q=%22https%3A%2F%2Fipfs.io%22), could we create a constant somewhere and import it instead?